doozerlib/cli/images_streams.py

```diff
@@ -608,7 +609,27 @@ def images_upstreampulls(runtime):
     print(yaml.dump(retdata, default_flow_style=False, width=10000))
 
 
-def reconcile_jira_issues(runtime, pr_links: Dict[str, str], dry_run: bool):
+def connect_issue_with_pr(pr: PullRequest.PullRequest, issue: str):
+    """
+    Aligns an existing Jira issue with a PR. Put the issue number in the title of the github pr.
+    Args:
+        pr: The PR to align with
+        issue: JIRA issue key
+    """
+    if issue in pr.title:  # the issue already in pr title
+        return
+    elif "OCPBUGS" in pr.title:  # another issue is in pr title, add comment
+        for comment in pr.get_issue_comments():
+            if issue in comment.body:
+                return  # an exist comment already have the issue
+        matches = re.findall(r'OCPBUGS-[0-9]+', pr.title)
+        pr.create_issue_comment(f"ART wants to connect issue [{issue}](https://issues.redhat.com/browse/{issue}) to this PR, \
+                                but found it is currently hooked up to {matches}. Please consult with #aos-art if it is not clear what there is to do.")
+    else:  # update pr title
+        pr.edit(title=f"{issue}: {pr.title}")
+
+
+def reconcile_jira_issues(runtime, pr_links: Dict[str, PullRequest.PullRequest], dry_run: bool):
```

Comment:
Parameter name `pr_links` is not appropriate.

Reply:
update the parameter pr_links to pr_map